spec/system/candidate_interface/references/candidate_request_references_continuous_applications_feature_enabled_spec.rb

```diff
@@ -24,6 +24,7 @@
     and_i_click_save_and_continue
     and_i_should_be_on_add_email_address_page
     and_the_back_link_should_point_to_the_add_name_page
+    and_i_click_save_and_continue
```

Comment:
I approved but this `and_i_click_save_and_continue` doesn't really have a place here.

Reply:
It does because it triggers the validations (which caused the bug)

Reply:
I know, but you're testing in a system spec that a 500 doesn't happen. Which in my opinion is not something that should be tested here.

This doesn't have a well defined logical flow:
```
and_i_click_save_and_continue
and_i_fill_the_email_address
and_i_click_save_and_continue
```

Reply:
If you want to do that, check for the expected error messages.

Reply:
Done the last suggested change.